src/nvim/indent.c

```diff
@@ -120,7 +120,7 @@ int set_indent(int size, int flags)
       // Count as many characters as we can use.
       while (todo > 0 && ascii_iswhite(*p)) {
         if (*p == TAB) {
-          tab_pad = (int)curbuf->b_p_ts - (ind_done % (int)curbuf->b_p_ts);
```

Comment:
wasn't all changes in `indent.c` related to the `curbuf->b_p_ts` change? (not saying the changes are wrong, but they are not related to this PR anymore?)